src/main/java/seedu/address/model/PdfConverter.java

```diff
@@ -65,19 +65,21 @@ public class PdfConverter {
 
     private PDPageContentStream contentStream;
 
-    /**
-     * Exports {@code key}'s task and score list in the form of a PDF file from this {@code Mathutoring}.
-     */
-    public PDDocument exportProgress(Student key) throws IOException {
-        requireAllNonNull(key);
+    public void setUp() throws IOException {
```

Comment:
setup is a single word, dont have the camelcase it.

Reply:
noted!